src/reuse/init.py

```diff
@@ -23,7 +23,8 @@ def prompt_licenses(out: IO[str] = sys.stdout) -> List[str]:
     """Prompt the user for a list of licenses."""
     first = _(
         "What license is your project under? "
-        "Provide the SPDX License Identifier."
+        "Provide the SPDX License Identifier. "
+        "See https://spdx.org/licenses/ for the list."
```

Comment:
```suggestion
        "See <https://spdx.org/licenses/> for the list."
```

The added brackets should make it easier to click in some terminals.

Reply:
Updated and checked that while the terminal I use (kitty) doesn't need this, it also isn't thrown off by it, so :+1: by me